RecoEgamma/EgammaIsolationAlgos/plugins/EgammaEcalRecHitIsolationProducer.cc

```diff
@@ -100,25 +90,16 @@ EgammaEcalRecHitIsolationProducer::EgammaEcalRecHitIsolationProducer(const edm::
   produces<edm::ValueMap<double>>();
 }
 
-EgammaEcalRecHitIsolationProducer::~EgammaEcalRecHitIsolationProducer() {}
-
-//
-// member functions
-//
-
 // ------------ method called to produce the data  ------------
 void EgammaEcalRecHitIsolationProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
   // Get the  filtered objects
-  edm::Handle<edm::View<reco::Candidate>> emObjectHandle;
-  iEvent.getByLabel(emObjectProducer_, emObjectHandle);
+  auto emObjectHandle = iEvent.getHandle(emObjectProducer_);
 
   // Next get Ecal hits barrel
-  edm::Handle<EcalRecHitCollection> ecalBarrelRecHitHandle;  //EcalRecHitCollection is a typedef to
-  iEvent.getByLabel(ecalBarrelRecHitProducer_.label(), ecalBarrelRecHitCollection_.label(), ecalBarrelRecHitHandle);
+  auto ecalBarrelRecHitHandle = iEvent.getHandle(ecalBarrelRecHitCollection_);
```

Comment:
it's nice to see this odd pattern go away.
apparently this is the only case in the active code, based on `git grep "getByLabel[^$]*label()[^$]*label()"`